From 1c1de41bfb1de89be886cfca6b8cf91e34d8a836 Mon Sep 17 00:00:00 2001
From: Francisco Boni <boboniboni@gmail.com>
Date: Tue, 11 Jan 2022 19:06:58 -0300
Subject: [PATCH 2/2] Disable powf check

---
 configure.ac | 9 ---------
 1 file changed, 9 deletions(-)

diff --git a/configure.ac b/configure.ac
index 39e6c61bc..ea0080493 100644
--- a/configure.ac
+++ b/configure.ac
@@ -234,15 +234,6 @@ struct dirent d_test;
 d_test.d_type = 0;
 }])],[AC_MSG_RESULT(yes);AC_DEFINE(DIRENT_HAS_D_TYPE,1,[struct dirent has d_type])],AC_MSG_RESULT(no))
 
-dnl TEST: Check for powf
-AC_MSG_CHECKING(for powf in libm);
-LIBS_BACKUP=$LIBS;
-LIBS="$LIBS -lm";
-AC_LINK_IFELSE([AC_LANG_PROGRAM([[#include <math.h>]],[[
-        powf(1.0f, 1.0f);
-]])], [AC_MSG_RESULT(yes)], [AC_DEFINE([DB_HAVE_NO_POWF],[1],[libm does not include powf])])
-LIBS=$LIBS_BACKUP
-
 dnl Look for clock_gettime, a DB_HAVE_CLOCK_GETTIME is set when present
 AH_TEMPLATE([DB_HAVE_CLOCK_GETTIME],[Determines if the function clock_gettime is available.])
 AC_SEARCH_LIBS([clock_gettime], [rt] , [found_clock_gettime=yes], [found_clock_gettime=no])
-- 
2.34.1

