From 18a880bcccea6a7be8f76e7814868c91e5bff3f1 Mon Sep 17 00:00:00 2001
From: Francisco Boni <boboniboni@gmail.com>
Date: Tue, 11 Jan 2022 19:06:58 -0300
Subject: [PATCH 2/2] Disable powf check

---
 configure.ac | 26 +++-----------------------
 1 file changed, 3 insertions(+), 23 deletions(-)

diff --git a/configure.ac b/configure.ac
index 39e6c61bc..1951797cc 100644
--- a/configure.ac
+++ b/configure.ac
@@ -234,15 +234,6 @@ struct dirent d_test;
 d_test.d_type = 0;
 }])],[AC_MSG_RESULT(yes);AC_DEFINE(DIRENT_HAS_D_TYPE,1,[struct dirent has d_type])],AC_MSG_RESULT(no))
 
-dnl TEST: Check for powf
-AC_MSG_CHECKING(for powf in libm);
-LIBS_BACKUP=$LIBS;
-LIBS="$LIBS -lm";
-AC_LINK_IFELSE([AC_LANG_PROGRAM([[#include <math.h>]],[[
-        powf(1.0f, 1.0f);
-]])], [AC_MSG_RESULT(yes)], [AC_DEFINE([DB_HAVE_NO_POWF],[1],[libm does not include powf])])
-LIBS=$LIBS_BACKUP
-
 dnl Look for clock_gettime, a DB_HAVE_CLOCK_GETTIME is set when present
 AH_TEMPLATE([DB_HAVE_CLOCK_GETTIME],[Determines if the function clock_gettime is available.])
 AC_SEARCH_LIBS([clock_gettime], [rt] , [found_clock_gettime=yes], [found_clock_gettime=no])
@@ -650,9 +641,6 @@ if test x$enable_x11 != xno; then
   fi
 fi
 
-dnl LIBRARY TEST: Fluidsynth support
-AC_CHECK_HEADER(fluidsynth.h,have_fluidsynth_h=yes,)
-
 dnl LIBRARY TEST: FreeType2
 AH_TEMPLATE(C_FREETYPE,[Define to 1 to enable freetype support])
 AC_ARG_ENABLE(freetype,AC_HELP_STRING([--disable-freetype],[Disable freetype support]),enable_freetype=$enableval,enable_freetype=yes)
@@ -1107,17 +1095,9 @@ if test x$enable_x11 != xno; then
   fi
 fi
 
-dnl FEATURE: Whether to use Fluidsynth
-AH_TEMPLATE(C_FLUIDSYNTH,[Define to 1 to enable libfluidsynth MIDI synthesis])
-AC_ARG_ENABLE(libfluidsynth,AC_HELP_STRING([--disable-libfluidsynth],[Disable libfluidsynth support]),enable_libfluidsynth=$enableval,enable_libfluidsynth=yes)
-if test x$enable_libfluidsynth = xyes; then
-    if test x$have_fluidsynth_lib = xyes -a x$have_fluidsynth_h = xyes ; then
-      LIBS="$LIBS -lfluidsynth"
-      AC_DEFINE(C_FLUIDSYNTH,1)
-    else
-      AC_MSG_WARN([libfluidsynth MIDI synthesis not available])
-    fi
-fi
+
+LIBS="$LIBS -lfluidsynth"
+AC_DEFINE(C_FLUIDSYNTH,1)
 
 dnl FEATURE: FFMPEG output support
 AH_TEMPLATE(C_AVCODEC,[Define to 1 to use FFMPEG libavcodec for video capture])
-- 
2.34.1

