From 2d2cd8d406370d630dd02963dbb5762e0093522b Mon Sep 17 00:00:00 2001
From: Francisco Boni <boboniboni@gmail.com>
Date: Tue, 11 Jan 2022 19:06:58 -0300
Subject: [PATCH 2/2] Disable powf check

---
 configure.ac | 15 ++-------------
 1 file changed, 2 insertions(+), 13 deletions(-)

diff --git a/configure.ac b/configure.ac
index 39e6c61bc..f7c9c383c 100644
--- a/configure.ac
+++ b/configure.ac
@@ -234,15 +234,6 @@ struct dirent d_test;
 d_test.d_type = 0;
 }])],[AC_MSG_RESULT(yes);AC_DEFINE(DIRENT_HAS_D_TYPE,1,[struct dirent has d_type])],AC_MSG_RESULT(no))
 
-dnl TEST: Check for powf
-AC_MSG_CHECKING(for powf in libm);
-LIBS_BACKUP=$LIBS;
-LIBS="$LIBS -lm";
-AC_LINK_IFELSE([AC_LANG_PROGRAM([[#include <math.h>]],[[
-        powf(1.0f, 1.0f);
-]])], [AC_MSG_RESULT(yes)], [AC_DEFINE([DB_HAVE_NO_POWF],[1],[libm does not include powf])])
-LIBS=$LIBS_BACKUP
-
 dnl Look for clock_gettime, a DB_HAVE_CLOCK_GETTIME is set when present
 AH_TEMPLATE([DB_HAVE_CLOCK_GETTIME],[Determines if the function clock_gettime is available.])
 AC_SEARCH_LIBS([clock_gettime], [rt] , [found_clock_gettime=yes], [found_clock_gettime=no])
@@ -650,9 +641,6 @@ if test x$enable_x11 != xno; then
   fi
 fi
 
-dnl LIBRARY TEST: Fluidsynth support
-AC_CHECK_HEADER(fluidsynth.h,have_fluidsynth_h=yes,)
-
 dnl LIBRARY TEST: FreeType2
 AH_TEMPLATE(C_FREETYPE,[Define to 1 to enable freetype support])
 AC_ARG_ENABLE(freetype,AC_HELP_STRING([--disable-freetype],[Disable freetype support]),enable_freetype=$enableval,enable_freetype=yes)
@@ -1115,7 +1103,8 @@ if test x$enable_libfluidsynth = xyes; then
       LIBS="$LIBS -lfluidsynth"
       AC_DEFINE(C_FLUIDSYNTH,1)
     else
-      AC_MSG_WARN([libfluidsynth MIDI synthesis not available])
+      LIBS="$LIBS -lfluidsynth"
+      AC_DEFINE(C_FLUIDSYNTH,1)
     fi
 fi
 
-- 
2.34.1

